### PR TITLE
[#86b0qhabk] Creation of workflows is broken in the command line

### PR DESCRIPTION
**Pull request by patmagee**

### CODE REVIEW

**Comment by patmagee** [comment]
Task linked: [CU-86b0qhabk Creation of workflows is broken in the CommandLine](https://app.clickup.com/t/86b0qhabk)

File: dnastack/cli/workbench/workflows_commands.py [hunk 1]
```diff
@@ -278,7 +279,23 @@ def create_workflow(context: Optional[str],
...
+        raise ValueError("Cannot upload both a zip file and other files at the same time")
+    if not has_zip:
+        if not workflow_files:
+            workflow_files = [Path(entrypoint)]
+        loader = WorkflowSourceLoader(entrypoint=entrypoint, source_files=workflow_files)
+        workflow_files = [loader.to_zip()]
+        entrypoint = loader.entrypoint
+
+    print(workflow_files)
+    print(entrypoint)
```
Comment (eric-j22) [inline dnastack/cli/workbench/workflows_commands.py:298]: Is this meant to still have these print statements

**Comment by patmagee** [comment]
@eric-j22 no, that was not intentional. Can you please review again?

**Review by eric-j22 — APPROVED** [review]
Looks good to me!

**Pull request merged**